### ISSUE TITLE
error while loading shared libraries: libgmp.so.10

### ISSUE THREAD

**Issue by simonh1000** [commercialhaskell/stack#2761]
### Stack version

```
$ stack --version
Version 1.2.0, Git revision 241cd07d576d9c0c0e712e83d947e3dd64541c42 (4054 commits) x86_64 hpack-0.14.0
```

I just installed Stack on an Ubuntu 16.04 LTS machine and used it successfully to compile some code.  However, I have been using pre-compiled binaries (of Elm 0.18 alpha) successfully before the installation of Slack and now get
```
error while loading shared libraries: libgmp.so.10
```
whenever I try to use the Elm CLI tools. I looked at https://github.com/commercialhaskell/stack/issues/967 but that seems to be about an old version of Linux whereas I am on a very new version.

I have no idea what to do next but can report the following
```
$ locate libgmp.so.10
/usr/lib/x86_64-linux-gnu/libgmp.so.10
/usr/lib/x86_64-linux-gnu/libgmp.so.10.3.0
```

I've even tried uninstall Stack but still get the errors.

**Comment by mgsloan** [comment]
So, if I read correctly, you didn't build elm with stack, but somehow installing stack caused these errors with elm?  Very curious!  Not sure why that would happen.

**Comment by mgsloan** [comment]
Closing as an old support ticket.

**Comment by KurtRMueller** [comment]
Was there no resolution to this issue?

**Comment by Martii** [comment]
Know this is old however I currently see a potential file ownership issue, at least currently in Ubuntu/Kubuntu 18.04 with:

``` sh-session
-rwxr-xr-x   1 1001 users   594864 Nov 30 05:55  libgmp.so.10
```

Related permission refs:
* https://forums.virtualbox.org/viewtopic.php?f=3&t=88080 *(always helpful when a sys mod doesn't take much time to answer that as well)
* https://www.virtualbox.org/ticket/16759 *(Partial solution for vbox which lead to this issue e.g. change ownership of `/usr` and `/usr/lib` to `root:root`... may resolve this projects issue if applied to the `.so`)*

Long story short... check the file permissions I think.
